apps/frontend/src/network/users/Research.tsx

```diff
@@ -78,6 +79,11 @@ const Research: React.FC<ResearchProps> = ({ user }) => {
               />
             </Frame>
           </Route>
+          <Route path={path + route.editSkills.template}>
+            <Frame title="Edit Expertise and resources">
```

Comment:
```suggestion
            <Frame title="Edit Expertise and Resources">
```

Reply:
I copy paste all copy but I think your correct 🤷

Reply:
This is not copy though is it, we make up the `<title>`s ourselves